biojava-structure/src/main/java/org/biojava/nbio/structure/io/LocalPDBDirectory.java

```diff
@@ -687,6 +690,14 @@ public File getLocalFile(String pdbId) {
 				for(String ex : getExtensions() ){
 					File f = new File(searchdir,prefix + pdbId.toLowerCase() + ex) ;
 					if ( f.exists()) {
+						// delete files that are too short to have contents
+						if( f.length() < MIN_PDB_FILE_SIZE ) {
+							boolean success = f.delete();
+							if( ! success) {
+								return null;
+							}
+							assert(!f.exists());
```

Comment:
I'd rather log a warning here. Assert is going to be ignored in normal situations

Reply:
That's intended. If the delete was successful then the file will not exist. This is just a sanity check during development to make sure delete() wasn't asynchronous or something.